internal/tasks/uploads_test.go

```diff
@@ -52,7 +53,7 @@ func TestDeleteAbandonedUploadWithZeroChunks(t *testing.T) {
 func TestDeleteAbandonedUploadWithOneChunk(t *testing.T) {
 	testDeleteUpload(t, func(sd keppel.StorageDriver, account models.Account) models.Upload {
 		data := "just some test data"
-		err := sd.AppendToBlob(account, testStorageID, 1, p2len(data), strings.NewReader(data))
+		err := sd.AppendToBlob(context.TODO(), account, testStorageID, 1, p2len(data), strings.NewReader(data))
```

Comment:
I would like to see `s.Ctx` being used here. Please amend `testDeleteUpload` to provide the `s test.Setup` variable to its callback.